### ISSUE TITLE
Error generating sourcemaps?

### ISSUE THREAD

**Issue by bertez** [stylecow/gulp-stylecow#1]
Hi, I'm using gulp-sourcemaps to generate inline sourcemaps from a sass file and gulp-stylecow to post-process the resulting css on my gulp task:

```js
gulp.task('css:dev', function() {
    return gulp.src(styleToProcess.input)
        .pipe(sourcemaps.init())
        .pipe(sass().on('error', sass.logError))
        .pipe(stylecow(merge({}, styleCowOptions, {
            code: 'normal'
        })))
        .pipe(sourcemaps.write())
        .pipe(gulp.dest(styleToProcess.output))
        .pipe(browserSync.stream());
});
``` 

The problem is that snippet is not generating sourcemaps right, but it I remove stylecow it is working fine.

The generated sourcemap is:
https://gist.github.com/bertez/f3ca6eeded96373d9c84e4ebdc238774

**Comment by bertez** [comment]
The sourcemap (decoded) generated without stylecow is this one:

https://gist.github.com/bertez/e2841c9b5c3be3ea64d541c1c571f34a

I think the problem is the `sources` key at the beggining of the sourcemap.

**Comment by oscarotero** [comment]
Interesting. I'll take a look. Thanks

**Comment by oscarotero** [comment]
This is a problem of stylecow-core that it's fixed in v2.4.1.
You have to do a `npm install` again and it should work fine.
Let me know whether you still having this problem. Cheers
